finance-tracker/app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class FriendshipsController < ApplicationController
+  def create
+  end
+
+  def destroy
+    friendship = current_user.friendships.find_by(friend_id: params[:id])
+    friendship.destroy
+    flash[:notice] = "Stopped following"
```

Comment:
How sure are you the `.destroy` happen without any error?
You doing it like this is:

```
1. destroy friendship
2. error in destroying friendship
3. Set flash notice to say "Stopped following". Then refresh page the data still tehre case of (2)
```